resources/build_resources.sh

```diff
@@ -24,7 +24,7 @@ function build_qt {
     $1 $2 > $UI_PYTHON_PATH/$3.py
     
     # replace PySide imports with tank.platform.qt and remove line containing Created by date
-    sed -i "" -e "s/from PySide import/from tank.platform.qt import/g" -e "/# Created:/d" $UI_PYTHON_PATH/$3.py
+    sed -i "" -e "s/\(from PySide import \(.*\)\)/try:\n    from tank.platform.qt import \2\nexcept ImportError:\n    \1/g" -e "/# Created:/d" $UI_PYTHON_PATH/$3.py
```

Comment:
Nice. Better than what I was thinking. Maybe Add an additional comment in here to explain why the `try`/`except`?